dashboard/modules/metrics/metrics_head.py

```diff
@@ -320,10 +325,29 @@ async def record_dashboard_metrics(self):
             SessionName=self._session_name,
         ).set(float(self._dashboard_proc.memory_full_info().uss) / 1.0e6)
 
+        # Report the max lag since the last export.
+        max_lag = max(self._event_loop_lag_samples)
+        self._event_loop_lag_samples = []
+        self._dashboard_head.metrics.metrics_event_loop_lag.labels(
+            ip=self._ip,
+            pid=self._pid,
+            Version=ray.__version__,
+            Component=self._component,
+            SessionName=self._session_name,
+        ).set(float(max_lag))
+
     async def run(self, server):
         self._create_default_grafana_configs()
         self._create_default_prometheus_configs()
 
+        def on_new_lag(lag_s):
+            # Record the lag. It's exported in `record_dashboard_metrics`
+            self._event_loop_lag_samples.append(lag_s)
```

Comment:
I think we don't need to maintain a list, just the max?

The reason is that I don't want to worry about potential OOM

Reply:
done